### ISSUE TITLE
The IFR can be estimated from the data

### ISSUE THREAD

**Issue by gustavdelius** [ImperialCollegeLondon/covid19model#80]
This issue is for information only. I want to share the observation that the infection fatality ratio, that is such a crucial input to the model and whose true value is so uncertain, can be estimated from the mortality data itself, and the result is surprising. The reason the IFR is a priori so uncertain is that what we can observe is only the CFR, the case fatality ratio. We do not know how many undetected infections there are for every detected case. The Imperial group have done their best to come up with good values for the IFR, but a large uncertainty remains.

As we describe at https://github.com/gustavdelius/covid19model/blob/master/covid19_IFR_report.pdf , if you put a broader prior distribution on the mean of the IFR, to acknowledge our uncertain knowledge, the posterior is actually quite peaked at a value that is an order of magnitude smaller than previously assumed, i.e., the model and the data taken together suggest that the virus is less deadly than assumed, but at the same time also more virulous.

The reason it is possible to extract information about the IFR from the mortality data is that the IFR determines the number of infections that are deduced from those mortalities, and those infections are assumed to lead to immunity, and the build-up of immunity over time lowers the R_t over time, which it turns out is detectable in the shape of the mortality curves. This is illustrated in the graph from the model with the broader prior below, where the curvature in R_t due to immunity is clearly visible.

![United_Kingdom_three_pannel_step_ifr-1491471-stanfit Rdata](https://user-images.githubusercontent.com/1340713/79733813-98c30880-82ed-11ea-83de-3ab70fd00445.png)

This observation that the data can be used to learn values for the IFR may be of particular value to people who are wanting to extend the model to new countries or regions and are not sure how to choose the IFR. 

You can play with this at https://github.com/gustavdelius/covid19model and I welcome suggestions and questions on the issue tracker there, https://github.com/gustavdelius/covid19model/issues . The report itself has been put on medRxiv.org. 

It is hopefully not necessary, but nevertheless I would like to stress that it is important to understand that these are model results, valid only in the context of this model, which may or may not adequately reflect the real world. Do not go out and break the shutdown based on this model.

**Comment by dsjoerg** [comment]
New York City saying hello.  Appreciate if you can update your analysis as seroprevalence results continue to roll in !

**Comment by gustavdelius** [comment]
Hi @dsjoerg, when you run the Imperial College model, with our broader prior to take the uncertainty in the prior knowledge of the IFR onto account, on today's data from the ECDC you get the following predictions for the immunity levels:

        countries                  value
          Austria   8.81% [2.19%-14.63%]
          Belgium  70.12% [26.83%-87.62%]
          Denmark  10.31% [2.41%-17.32%]
           France  40.59% [11.48%-61.24%]
          Germany  10.29% [2.39%-17.92%]
           Greece   1.30% [0.33%-2.21%]
            Italy  41.79% [12.19%-62.04%]
      Netherlands  35.66% [9.34%-56.96%]
           Norway   5.90% [1.46%-10.35%]
         Portugal  11.08% [2.73%-18.64%]
            Spain  57.34% [15.51%-80.44%]
           Sweden  60.36% [20.45%-86.62%]
      Switzerland  20.76% [5.05%-33.22%]
   United_Kingdom  42.24% [12.23%-63.03%]

However it is my understanding that the Imperial College group is working on a revision of their model. I expect that they will take our message on board that a broader prior for the IFR would be more realistic (@flaxter, is that right?), but I would not be surprised if their improved model will lead to lower predictions for the immunity than the current version. Until we see the improved version, take the above results with a big grain of salt.

By the way, our paper is now on medRxiv: https://doi.org/10.1101/2020.04.19.20071811

**Comment by dsjoerg** [comment]
I'm surprised that you call it "immunity" rather than "cumulative % infected".   They are not the same, right?  And the model is predicting the latter?

And, thank you for the pointer and for the preprint. :)

**Comment by gustavdelius** [comment]
@dsjoerg You are right that we do not know whether having been infected leads to immunity in the real world. What the model actually detects in the data is immunity. While it was assumed in the original model that having been infected and being immune is the same thing, I don't think the conclusions depend very much on that. We also ran the model assuming that 20% of the population were immune even before ever being infected, and the predicted current immunity did not change very much.

**Comment by dsjoerg** [comment]
This may be a noob question, but would a diminishment in R_t due to other effects (such as people learning to reduce contact to protect themselves) be interpreted/confounded by your model as growing immunity?  And thus high infection / low IFR?  When in fact it's a reduction of R_t unrelated to those things?

**Comment by gustavdelius** [comment]
Yes, that was our first worry. It is a particular feature of this model that R_t is assumed to be a step function, i.e., that each intervention achieves its full strength on the day it is implemented. We therefore investigated what would happen if the strength increases over a period. See section 3.3.3 of our report. This changed quite dramatically how the model estimated the strength of the effects of the various interventions, but did not change the estimate for the IFR in a noticeable way. However that does not mean that some other model of how the strength of interventions changes over time might not lead to a lower estimate of the IFR. It would be good if people would suggest different models that we could check out.

**Comment by zach-hensel** [comment]
41.79% of all of Italy being infected and 60.36% of all of Sweden being infected are not credible given regional distributions of deaths, hospitalizations, ICU, cases... e.g. 9.5% of Swedish population in Stockholm and 53% of deaths. If 60% of all of Sweden is infected and deaths occur in the same place as infections, 335% of Stockholm has been infected. Similarly there are over 10-fold differences in statistics between regions of Italy.

**Comment by becare-rocket** [comment]
Both models (Gustav's and the ICL model) should be recalibrated when broad seroprevalence surveys are completed for the respective countries.
